docs/java-rest/high-level/ml/get-calendars.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[java-rest-high-x-pack-ml-get-calendars]]
+=== Get Calendars API
+Retrieves one or more calendar objects.
+It accepts a `GetCalendarsRequest` and responds
+with a `GetCalendarsResponse` object.
+
+[[java-rest-high-x-pack-ml-get-calendars-request]]
+==== Get Calendars Request
+
+By default a `GetCalendarsRequest` with no calendar Id set will return all
+calendars, using the literal `_all` also returns all calendars.
```

Comment:
Might sound better with a full stop rather than a comma after "calendars".  Then "Using the literal `_all`..." as a new sentence.